docs/source/guides/contributing.md

```diff
@@ -17,12 +17,12 @@ specific to Inko's version manager should be reported in the project
 [inko-lang/ivm](https://github.com/inko-lang/ivm/issues). Here are a few of
 our projects:
 
-| Project            | Description               | Issue tracker
-|:-------------------|:--------------------------|:----------------------------
-| inko-lang/inko     | The main project for Inko | <https://github.com/inko-lang/inko/issues>
-| inko-lang/ivm      | Inko's version manager    | <https://github.com/inko-lang/ivm/issues>
-| inko-lang/inko.vim | Vim integration for Inko  | <https://github.com/inko-lang/inko.vim/issues>
-| inko-lang/website  | The Inko website          | <https://github.com/inko-lang/website/issues>
+| Project            | Description               | Issue tracker                                  |
+| :----------------- | :------------------------ | :--------------------------------------------- |
+| inko-lang/inko     | The main project for Inko | <https://github.com/inko-lang/inko/issues>     |
+| inko-lang/ivm      | Inko's version manager    | <https://github.com/inko-lang/ivm/issues>      |
+| inko-lang/inko.vim | Vim integration for Inko  | <https://github.com/inko-lang/inko.vim/issues> |
+| inko-lang/website  | The Inko website          | <https://github.com/inko-lang/website/issues>  |
```

Comment:
These changes are unnecessary, as they're not related to the rest and the trailing `|` isn't needed in Markdown tables.